### ISSUE TITLE
Delete items from the Library

### ISSUE THREAD

**Issue by pieterlechner** [fer0n/Unwatched#25]
Is there any way to delete something from the Library. I may be missing something but I don't see a way to do it. Sorry if this is a stupid question.

**Comment by fer0n** [comment]
There's currently no way to delete individual videos. The only way to currently have them deleted is when you unsubscribe from a channel from the subscription detail page and then leave the page. In that case all videos not marked as watched/bookmarked/queued will be deleted.

The problem is if you're still subscribed and the video is one of the last 15 from that channel, it'll simply reappear when you refresh, although there's probably a way to handle this edge case.

What exactly is your use case for deleting videos?

**Comment by pieterlechner** [comment]
Thanks for getting back to me so quickly. I often add a single video without subscribing to the channel. I want to see if it’s worth subscribing or it might just be a video I want to watch more than once. But at some point I’m done with it and don’t plan on subscribing.

But it’s easy enough to subscribe and then unsubscribe which I hadn’t thought of before. So that solves my issue. Thank you and I really enjoy using Unwatched. 

Pieter

**Comment by fer0n** [comment]
I'm assuming you're aware that you can clear the video from your queue/inbox? It's not taking up a huge amount of space to leave the video inside the "all videos" section, but I guess there's an desire to keep things clean

**Comment by pieterlechner** [comment]
Yes and I do clear them, just in a few cases I’d just as soon they were gone and you let me know how to do that so all is well.
